src/components/Drawer.js

```diff
@@ -0,0 +1,28 @@
+import React, { useRef } from "react";
```

Comment:
No nedd for ref. Pleae Delete all related code

Reply:
done